extensions/amp-story/amp-story-ads.md

```diff
@@ -0,0 +1,176 @@
+# Advertising support in AMP Stories 
+
+## AMP Story Ad Principles
+
+The current ad ecosystem of banners and boxes doesn't integrate well with the rest of the content around it. Ads feel out of place, are slow and interruptive and don't feel native to the rest of the story experience.
+
+We'd like to establish the following principles for Story Ads:
+
+* *Visual first*:  Inviting, bold, context-driven invitation state.
+* *Native*: The ad page has the same dimensions as an organic story page. 
+* *Same interaction model*: User can continue to the next screen just like they would with an organic story page.
+* *Fast*: The ad never appears to a user in a half-loaded state. 
+
+As a result, one key difference from regular web pages is that the AMP stories runtime determines the right placement of the ad page amidst the AMP story. These mechanics are explained as part of [amp-story-auto-ads](./amp-story-auto-ads.md).
+
+
+## Ad formats
+There are two type of ad formats supported as part of AMP Stories:
+
+* **Single page ad** : Where the ad appears as a single page inside of an AMP story.
+* **Sponsored story ad**: Where the ad is a stand-alone multi-page story.
+
+### Single page ad 
+
+In-line with the principles, a single page ad appears in between organic story content as a full page. 
+Single Page Ads have a predefined set of call to action buttons and those call to action buttons take a web landing page URL where the user is navigated to, on click.
+
+![Story Page Ad](img/story-page-ad.png)
+
+### Sponsored story ad 
+When publishing sponsored story ads, use the `<amp-story ad>` tag instead of `<amp-story>`, so platforms can surface these stories appropriately. Since a sponsored story ad is just a URL on the web, you can also send user traffic to a sponsored story ad from the CTA URL button of a single page ad. 
+
+![Sponsored Story Ad](img/sponsored-story-ad.png)
```

Comment:
This image uses the term "STAMP" which isn't familiar to readers.  Please redo the image to use "AMP story" or "Story page"

Reply:
done